pkg/kudoctl/cmd/install/install.go

```diff
@@ -69,24 +72,42 @@ func validate(args []string, options *Options) error {
 // getPackageCRDs tries to look for package files resolving the operator name to:
 // - a local tar.gz file
 // - a local directory
+// - a url to a tar.gz
 // - a operator name in the remote repository
 // in that order. Should there exist a local folder e.g. `cassandra` it will take precedence
 // over the remote repository package with the same name.
-func getPackageCRDs(name string, options *Options, repository repo.Repository) (*repo.PackageCRDs, error) {
+func getPackageCRDs(name string, options *Options, repository repo.Repository) (*bundle.PackageCRDs, error) {
```

Comment:
Could we make this public?

Reply:
I've been reducing all things to be private until needed to be public.. is there are reason to make it public?